advanced/scanningArm/scanningArmDemoRunner.m

```diff
@@ -0,0 +1,20 @@
+function scanningArmDemoRunner( )
+
+    % Close any previously opened figures
+    close all;
+
+    % Make sure libraries and paths are loaded appropriately
+    localDir = fileparts(mfilename('fullpath'));
+    run(fullfile(localDir, 'startup.m'));
+
+    % Start the demo
+    while true
+        try
+            scanningArmRaster();
```

Comment:
If it gets executed one folder up, it finds the higher `startup.m` and searches for the `hrdf` file in the wrong spot.

Probably not worth worrying about though